metadata_backend/api/handlers/submission.py

```diff
@@ -23,7 +23,7 @@ class SubmissionAPIHandler(RESTAPIIntegrationHandler):
     def _make_discovery_url(self, obj_data: Dict) -> str:
         """Make an url that points to a discovery service."""
         # TODO: Proper URL creation when metax is disabled
```

Comment:
might want to have this `TODO` as an issue not here

Reply:
Yes, but not on this change. I created issue https://github.com/CSCfi/metadata-submitter/issues/571